cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/CompileCommand.java

```diff
@@ -207,9 +207,9 @@ public void execute() {
         
             targetPath = this.sourceRootPath.resolve(ProjectDirConstants.TARGET_DIR_NAME);
         } else {
-            throw LauncherUtils.createLauncherException("invalid ballerina source path, it should either be a module " +
-                                                        "name in a ballerina project or a file with a \'" +
-                                                        BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension.");
+            throw LauncherUtils.createLauncherException("invalid Ballerina source path. Either a module " +
+                                                        "name in a Ballerina project or a file with a \'" +
+                                                        BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension is expected.");
```

Comment:
```suggestion
                                                        BLangConstants.BLANG_SRC_FILE_SUFFIX + "\' extension.");
```

Reply:
It is correct to have "is expected" at the end of the sentence.